app/templating/summary/item.py

```diff
@@ -0,0 +1,23 @@
+from app.utilities.factory import Factory
+
+
+class Item(object):
+    def __init__(self, schema, state):
+        self.schema = schema
+        self.state = state
+        self.question = self.schema.title or self.schema.answers[0].label
+        self.link = self.schema.container.container.id + '#' + self.schema.id
+        if len(self.schema.answers) > 0:
```

Comment:
Either a comment or move this to a function with a good descriptive name as not clear what these two lines are doing and why